src/audio/codec_adapter/codec_adapter.c

```diff
@@ -46,17 +46,17 @@ static struct comp_dev *codec_adapter_new(const struct comp_driver *drv,
 	struct sof_ipc_comp_process *ipc_codec_adapter =
```

Comment:
This commit is already added in PR #3999 - https://github.com/thesofproject/sof/pull/3999/commits/c152db791c72bab53138570252982420f8cf3956

Reply:
@iuliana-prodan Initially I added it only in this PR hoping it will get merge first. Anyhow, it doesn't matter that a commit is in 2 PR, git merge should easily take care of this.

Reply:
@iuliana-prodan removed the commit from this PR, so that we eliminate any confusion.

Reply:
I think that there might be some merge conflicts when applied on `main`. So, is safer this way.